src/IO/H5/VolumeData.hpp

```diff
@@ -25,21 +26,32 @@ namespace h5 {
  * The volume data inside the subfile can be of any dimensionality greater than
  * zero. This means that in a 3D simulation, data on 2-dimensional surfaces are
  * written as a VolumeData subfile. Data can be written using the
- * `insert_tensor_data()` method. An integral observation id is used to keep
+ * `write_volume_data()` method. An integral observation id is used to keep
  * track of the observation instance at which the data is written, and
  * associated with it is a floating point observation value, such as the
  * simulation time at which the data was written. The observation id will
  * generally be the result of hashing the temporal identifier used for the
  * simulation.
  *
  * The data stored in the subfile are the tensor components passed to the
```

Comment:
Please make a note that coordinates (in any frame) must be written as tensors as well in order for the data to be viewable in, e.g. ParaView, VisIt, etc.